### PR TITLE
add rewards distributor to pool

### PR DESCRIPTION
**Pull request by jmzwar**

### CODE REVIEW

File: protocol/synthetix/test/integration/modules/core/RewardsManagerModule.test.ts [hunk 1]
```diff
@@ -41,6 +41,15 @@ describe('RewardsManagerModule', function () {
...
     RewardDistributor = await factory.connect(owner).deploy();
 
+    console.log('System core address', systems().Core.address);
+
+    console.log(
+      'Collateral address rewards collateral',
+      Collateral.address,
+      'Collateral address core system',
+      collateralAddress()
+    );
```
Comment (bachstatter) [inline protocol/synthetix/test/integration/modules/core/RewardsManagerModule.test.ts:51]: Remove some logs

File: protocol/synthetix/subgraph/tests/core.test.ts [hunk 2]
```diff
@@ -634,6 +634,9 @@ describe('core tests', () => {
   test('handleRewardsDistributed', () => {
     // Needs to be here because of Closures
     const now = new Date(1668448739566).getTime();
+    const newPoolEvent = createPoolCreatedEvent(1, address, now, now - 1000);
+    handlePoolCreated(newPoolEvent);
```
Comment (bachstatter) [inline protocol/synthetix/subgraph/tests/core.test.ts:638]: Do you want to add some asserts to ensure the pool is assigned correctly to the rewards distributor too

File: markets/perps-market/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@synthetixio/perps-market",
-  "version": "3.2.0",
+  "version": "3.2.1-0",
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply markets/perps-market/package.json:3]: No idea, didn't edit this file myself

File: protocol/synthetix/subgraph/tests/core.test.ts (hunk 2 above)
Comment (jmzwar) [reply protocol/synthetix/subgraph/tests/core.test.ts:638]: Yeah will do

**Pull request merged**